packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -203,10 +205,12 @@ import 'basic_types.dart';
 @immutable
 class TextStyle extends Diagnosticable {
   /// Creates a text style.
+  ///
+  /// The [package] argument must be non-null if the font family is defined in a
+  /// package. See the documentation for the [TextStyle] class itself for details.
```

Comment:
Rather than "See the documentation for the [TextStyle] class itself for details." I would just say something like "The package argument is combined with the fontFamily argument to set the fontFamily property." or some such.

Also, prefer backticks over `[]` for arguments that don't have matching properties.